lms/djangoapps/teams/tests/test_views.py

```diff
@@ -129,7 +129,7 @@ def test_query_counts(self):
         team.add_user(self.user)
 
         # Check the query count on the dashboard again
-        with self.assertNumQueries(19):
+        with self.assertNumQueries(20):
```

Comment:
@nasthagiri 
Its the https://github.com/edx/edx-platform/blob/master/lms/djangoapps/teams/serializers.py#L61
that is using UserReadOnlySerializer